bin/sirius-script-rabpm-mngr.py

```diff
@@ -102,15 +111,17 @@ def main():
 
     parser_reset = subparsers.add_parser("rffe-reset",
                                          help="Reset RFFE")
-    parser_reset.add_argument("-v", "--vslot", required=True,
-                              help="Virtual slot number to reset(calculated as "
+    parser_reset.add_argument("-v", "--vslots", nargs="+", type=int, default=list(range(11, 24)),
+                              help="Virtual slot numbers to reset(calculated as "
                               "(2*physical_slot-1) or (2*physical_slot))")
     add_rack_arg(parser_reset)
 
     parser_boot_from_flash = subparsers.add_parser(
         "afc-reset", help="Boot the AFC from flash memory")
-    parser_boot_from_flash.add_argument("-s", "--slot", required=True,
-                                        help="Slot number")
+    parser_boot_from_flash.add_argument("-s", "--slots", nargs="+", type=int, default=list(range(1, 13)),
+                                        help="Slot numbers")
+    parser_boot_from_flash.add_argument("--all", choices=['timing', 'fofb', 'bpm'],
+                                        help="Reset a batch of specified afc devices (timing, fofb or bpm)")
```

Comment:
I think the argument "--all" has a misleading name. It should mean "only apply to this type of device (timing, fofb or bpm)". It's more like a convenient preset for "--slots" than anything else.

Maybe "--device-type" is a better name? @anacso17, what's your opinion on that? I think this is related to your review comment, as I understand this is the source of the confusion.